charts/ps-db/values.yaml

```diff
@@ -0,0 +1,189 @@
+  # schedulerName: mycustom-scheduler
+  annotations: {}
+    # iam.amazonaws.com/role: role-arn
+  labels: {}
+    # rack: rack-22
+  nodeSelector: {}
+    # disktype: ssd
+  tolerations: []
+    # - key: "node.alpha.kubernetes.io/unreachable"
+    #   operator: "Exists"
+    #   effect: "NoExecute"
+    #   tolerationSeconds: 6000
+  podDisruptionBudget:
+    # only one of maxUnavailable or minAvaliable can be set
+    maxUnavailable: 1
+    # minAvailable: 0
+  # A custom Kubernetes Security Context for a Container to be used instead of the default one
+  # containerSecurityContext:
+  #   privileged: false
+  # A custom Kubernetes Security Context for a Pod to be used instead of the default one
+  # podSecurityContext:
+  #   fsGroup: 1001
+  #   supplementalGroups:
+  #   - 1001
+  # serviceAccountName: percona-server-mysql-operator-workload
+  # sidecars:
+  # - name: noop-memory
+  #   image: busybox
+  #   command: ["sleep", "30d"]
+  #   volumeMounts:
+  #   - name: "memory-vol"
+  #     mountPath: "/var/log/app/memory"
+  #   resources:
+  #     requests:
+  #       memory: 16M
+  # - name: noop-pvc
+  #   image: busybox
+  #   command: ["sleep", "30d"]
+  #   volumeMounts:
+  #   - name: "pvc-vol"
+  #     mountPath: "/var/log/app/pvc"
+  # sidecarVolumes:
+  # - name: "memory-vol"
+  #   emptyDir:
+  #     medium: "Memory"
+  # sidecarPVCs:
+  # - name: pvc-vol
+  #   spec:
+  #     resources:
+  #       requests:
+  #         storage: 1G
+
+orchestrator:
+  image:
+    repository: percona/percona-server-mysql-operator
+    tag: 0.1.0-orchestrator
+  imagePullPolicy: Always
+  imagePullSecrets: []
+  # - name: private-registry-credentials
+
+  size: 1
+
+  resources:
+    requests:
+      memory: 128M
+    limits:
+      memory: 256M
+
+  affinity:
+    antiAffinityTopologyKey: "kubernetes.io/hostname"
+
+  volumeSpec:
+    pvc:
+      resources:
+        requests:
+          storage: 1G
+
+  # runtimeClassName: image-rc
+  # priorityClassName: high-priority
+  # schedulerName: mycustom-scheduler
+  annotations: {}
+    # iam.amazonaws.com/role: role-arn
+  labels: {}
+    # rack: rack-22
+  nodeSelector: {}
+    # disktype: ssd
+  tolerations: []
+    # - key: "node.alpha.kubernetes.io/unreachable"
+    #   operator: "Exists"
+    #   effect: "NoExecute"
+    #   tolerationSeconds: 6000
+  podDisruptionBudget:
+    # only one of maxUnavailable or minAvaliable can be set
+    maxUnavailable: 1
+    # minAvailable: 0
+  # A custom Kubernetes Security Context for a Container to be used instead of the default one
+  # containerSecurityContext:
+  #   privileged: false
+  # A custom Kubernetes Security Context for a Pod to be used instead of the default one
+  # podSecurityContext:
+  #   fsGroup: 1001
+  #   supplementalGroups:
+  #   - 1001
+  # serviceAccountName: percona-server-mysql-operator-workload
+
+pmm:
+  enabled: false
+  image:
+    repository: percona/pmm-client
+    tag: 2.25.0
+  imagePullPolicy: Always
```

Comment:
why so old pmm version?

Reply:
we use this version for 0.1.0

Reply:
I will change the chart for 0.2.0, so it will have 2.28.